test/e2e/flowaggregator_test.go

```diff
@@ -754,10 +795,17 @@ func getUnit64FieldFromRecord(t *testing.T, record string, field string) uint64
 // received all the expected records for a given flow with source IP, destination IP
 // and source port. We send source port to ignore the control flows during the
 // iperf test.
-func getCollectorOutput(t *testing.T, srcIP, dstIP, srcPort string, timeStart time.Time, checkAllRecords bool) (string, []string) {
+func getCollectorOutput(t *testing.T, srcIP, dstIP, srcPort string, timeStart time.Time, checkAllRecords bool, isRestart bool) (string, []string) {
 	var collectorOutput string
 	var recordSlices []string
-	err := wait.PollImmediate(500*time.Millisecond, aggregatorInactiveFlowRecordTimeout, func() (bool, error) {
+	timeout := aggregatorInactiveFlowRecordTimeout
+	// When flow aggregator gets restarted, flow exporter will close the connection firstly by a separate goroutine.
+	// Next time when sending record, flow exporter will try to send on closed connection, reset exporting process
+	// and then send it in the next check. Therefore, one more second is added.
+	if isRestart {
+		timeout = aggregatorInactiveFlowRecordTimeout + time.Second
```

Comment:
Any reason behind the additional one second? In addition, it would be useful if we have a constant with a separate name like `restartFlowAggregatorTimeout` or something along those lines.

Reply:
Makes sense. Updated.